testing/python/metafunc.py

```diff
@@ -220,6 +220,53 @@ def func(x, y): pass
         assert metafunc._calls[0].params == dict(x=1,y=2, unnamed=1)
         assert metafunc._calls[1].params == dict(x=1,y=3, unnamed=1)
 
+    def test_parametrize_indirect_list(self):
+        def func(x, y): pass
+        metafunc = self.Metafunc(func)
+        metafunc.parametrize('x, y', [('a', 'b')], indirect=['x'])
+        assert metafunc._calls[0].funcargs == dict(y='b')
```

Comment:
This is not your problem or mistake, but it seems a bit yuck that _calls is protected...